examples/custom-reducer/package.json

```diff
@@ -38,6 +38,7 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.0",
+    "babel-plugin-inline-json-import": "^0.2.1",
```

Comment:
need to bring this back to read package.json version value

Reply:
same as above

Reply:
related to the above one